src/governance.cpp

```diff
@@ -265,7 +266,6 @@ void CGovernanceManager::CheckOrphanVotes(CGovernanceObject& govobj, CGovernance
             fRemove = true;
         }
         else if(govobj.ProcessVote(NULL, vote, exception)) {
```

Comment:
Could probably also remove `fRemove` and rewrite/squash both `if` blocks here to smth like:
```
if(pairVote.second < nNow || govobj.ProcessVote(NULL, vote, exception)) {
    mapOrphanVotes.Erase(nHash, pairVote);
}
```